dependencies.yaml

```diff
@@ -309,10 +309,10 @@ dependencies:
 
   # golangci-lint
   - name: "golangci-lint"
-    version: 1.42.0
+    version: 8d0a2ad834fe92f69fe69918a68b0c141f0ef97980e6bba84b72bdd7d58fc3b5
     refPaths:
     - path: hack/verify-golangci-lint.sh
-      match: VERSION=v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?
+      match: IMAGE_SHA=sha256:(8d0a2ad834fe92f69fe69918a68b0c141f0ef97980e6bba84b72bdd7d58fc3b5)
```

Comment:
nit: here should be a regex

Reply:
this is a regex that matches only 1 string.